components/odh-notebook-controller/controllers/notebook_webhook.go

```diff
@@ -354,6 +354,12 @@ func (w *NotebookWebhook) Handle(ctx context.Context, req admission.Request) adm
 		if err != nil {
 			return admission.Errored(http.StatusInternalServerError, err)
 		}
+
+		// Mount Secret ds-pipeline-config
```

Comment:
Should we have a watch for DSPAs, similar as there is configmap watch in

https://github.com/opendatahub-io/kubeflow/blob/4ec227fe858c7848997c3c068441c961fab6c59f/components/odh-notebook-controller/controllers/notebook_controller.go#L484-L487
?